google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -122,8 +122,10 @@ appengine.stop.modules.version.canceled.message=Undeploy Canceled
 appengine.deployment.error.during.staging=Deployment failed due to staging failing with exit code: {0}.
 appengine.deployment.exception.during.staging=Deployment failed due to an exception while staging the project.
 appengine.deployment.error.staging.yaml=The specified app.yaml configuration file does not exist or is not a valid file.
-appengine.deployment.error.staging.dockerfile=The specified Dockerfile configuration file does not exist or is not a valid file.
-appengine.deployment.error.staging.gotosettings=Set a valid file in Module Settings or use another one.
+appengine.deployment.error.staging.dockerfile=There is no Dockerfile in specified directory or it is not a valid file.
```

Comment:
see my UX doc for proposed updates. perhaps those changes can be anticipated in this PR.

https://docs.google.com/document/d/1rE4Ir9UnRXoLnRxxxqF63KmBL8xkl_W1KOwqUwdtM6Y

I specifically do not have the "valid file" part because we don't do anything intelligent with the Dockerfile yet.